Formula/mypaint-brushes.rb

```diff
@@ -0,0 +1,29 @@
+class MypaintBrushes < Formula
+  desc "Brushes used by MyPaint and other software using libmypaint"
+  homepage "https://github.com/mypaint/mypaint-brushes"
+  url "https://github.com/mypaint/mypaint-brushes/archive/v1.3.0.tar.gz"
+  sha256 "704bb6420e65085acfd7a61d6050e96b0395c5eab078433f11406c355f16b214"
+  license "CC0-1.0"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
+  depends_on "libmypaint"
+
+  def install
+    ENV["ACLOCAL"] = "aclocal"
+    ENV["AUTOMAKE"] = "automake"
```

Comment:
Is this necessary?

Reply:
Unfortunately yes.

Reply:
Under what names does it look for `aclocal` and `automake` without this?

Reply:
`./autogen.sh: line 135: aclocal-1.13: command not found`

and

`./autogen.sh: line 113: automake-1.13: command not found`